packages/api-client/src/layouts/Modal/ApiClientModal.vue

```diff
@@ -44,12 +44,12 @@ onBeforeUnmount(() => {
   <div
     v-show="modalState.open"
     class="scalar">
-    <div className="scalar-container">
+    <div className="scalar-container z-overlay">
       <div className="scalar-app scalar-client scalar-app-layout">
         <RouterView key="$route.fullPath" />
       </div>
       <div
-        class="scalar-app-exit"
+        class="scalar-app-exit -z-1"
```

Comment:
any reason to have the z-overlay here (out of curiosity)? it seems like the `-z-1` is making it

Reply:
Should we use another Tailwind preset instead of a number here?

Reply:
can also just add the `z-overlay` as a `scalar-client` class sibling so we don't have to?

Reply:
We want to redefined the stacking context for `scalar-app-exit` so it gets moved behind `scalar-client` (which inherently has a `z-index` of 0 in the stacking context) but not behind other elements on the page. 

If we don't make it an overlay there then theres a risk of the app exit getting put behind other high z-index elements on the page the modal is embedded in.